modules/ROOT/pages/index.adoc

```diff
@@ -82,6 +82,112 @@ Monitor APIs and endpoints at scheduled intervals. Receive reports about whether
 
 See xref:api-functional-monitoring::afm-in-anypoint-platform.adoc[API Functional Monitoring in Anypoint Monitoring].
 
+== Anypoint Monitoring Features
+
+Anypoint Monitoring is the standard method of monitoring Mule application and API performance, logging, and troubleshooting to reduce the mean time to identify and resolve issues. While all customers receive fundamental monitoring capabilities with their subscription, the Titanium tier offers greater storage, guaranteed resources, and advanced capabilities that allow for greater control and flexibility.
+
+|===
+| Feature | Gold/Platinum subscription features | Titanium subscription features
+
+|Dedicated infrastructure for data processing and storage
+|No
+|Yes
+
+|Custom Metrics
+|No
+|Yes
+
+3+| *Built-in Application Dashboards*
+|More than 80 out-of-the-box charts covering metrics in the following categories: Overview, Inbound, Outbound, Performance, Failures, JVM, Infrastructure
+
+|Yes
+|Yes
+
+|Advanced metrics for Inbound and Outbound charts
+|No
+|Yes
+
+|Flows charts
+|No
+|Yes
+
+|Connectors charts
+|No
+|Yes
+
+|Anypoint Visualizer Application Network integration
+|No
+|Yes
+
+|xref:monitoring::performance-and-impact.adoc#data-retention-and-aggregation[Data retention] for more than 30 days
+|No
+|Yes
+
+3+| *Built-in API Dashboards*
+
+|16 out-of-the-box charts for metrics covering the following categories: Overview, Requests, Failures, Performance, Client Application 
+|Yes
+|Yes
+
+|xref:monitoring::performance-and-impact.adoc#data-retention-and-aggregation[Data retention] for more than 30 days
+|No
+|Yes
+
+3+| *Custom Dashboards*
+
+|Basic mode
+|Yes
+|Yes
+
+|Advanced mode
+|Yes, with limited functionality
+|Yes
+
+|Advanced mode with additional features
+|No
+|Yes
+
+3+| *Alerts*
+
+|Basic alerts
+|Yes
+|Yes, with xref:performance-and-impact.adoc#data-retention-and-aggregation[higher limits]
+
+|Advanced alerts
+|No
+|Yes
+
+3+| *Log Management*
+
+|Log search
+|No
+|Yes
+
+|Log points
+|No
+|Yes
+
+|Raw data
+|No
+|Yes
+
+3+| *Additional Anypoint Monitoring features*
+
+|Anypoint Monitoring Archive API
+|No
+|Yes
+
+|xref:api-functional-monitoring[API Functional Monitoring] +
+Note that API Functional Monitoring is not part of Anypoint Monitoring.
```

Comment:
Style Ref. has this:
The UI for API Functional Monitoring is the Functional Monitoring feature of Anypoint Monitoring.
Is this incorrect?

Reply:
@sarathecoubian  Let's not say Functional Monitoring is not part of Anypoint Monitoring. It's very confusing to the customers as the feature is linked in Anypoint Monitoring, and it should be part of Anypoint Monitoring. If the doc is written elsewhere, I suggest we deal with that separately. I'd like to see the doc moved over to Anypoint Monitoring. Again, we can deal with that seprately.

Reply:
```suggestion
```